Xamarin.Forms.sln

```diff
@@ -2572,6 +2574,58 @@ Global
 		{140BC260-8B15-4D3A-B1B0-DDD8072918CC}.Release|x86.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|Any CPU.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|Any CPU.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|ARM.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|ARM.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|iPhone.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|iPhone.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|iPhoneSimulator.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|iPhoneSimulator.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|Templates.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|Templates.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|x64.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|x64.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|x86.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Ad-Hoc|x86.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|Any CPU.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|Any CPU.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|ARM.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|ARM.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|iPhone.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|iPhone.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|iPhoneSimulator.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|iPhoneSimulator.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|Templates.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|Templates.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x64.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x64.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x86.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x86.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|ARM.Build.0 = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|iPhone.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|iPhoneSimulator.ActiveCfg = Debug|Any CPU
```

Comment:
Remove from iphone awn iphone simulator  build for release and debug

Reply:
I  think it's OK, I removed all lines with build.0 or Deploy.0 for debug and release.

Reply:
yeah it's working